File: Easy/two-sum/two-sum.c

#include <stdio.h>
#include <stdlib.h>
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int *twoSum(int *nums, int numsSize, int target, int *returnSize)
{
    int i, j,*returnArr;
    *returnSize=2;
    returnArr=(int *)malloc(sizeof(int)*2);
    for (i = 0; i < numsSize - 1; i++)
    {
        for (j = i+1; j < numsSize; j++)
        {
            if(nums[i]+nums[j]==target){
     
                *returnArr++=i;
                *returnArr--=j;
                return returnArr;
            }
        }
    }
    return NULL;
}

File: Easy/two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numLeng=len(nums)
        for i in range(0,numLeng-1):
            for j in range(i+1,numLeng):
                if(nums[i]+nums[j]==target):
                    return [i,j]
s=Solution()
s.twoSum([3,2,4],6)